src/components/List.js

```diff
@@ -36,29 +45,49 @@ const List = ({ token }) => {
     <div>
       <h1>List</h1>
 
-      {docs.length === 0 && (
+      {docs.length === 0 ? (
         <section>
           <p>Your shopping list is currently empty.</p>
           <a href="/add-item">Add an Item</a>
         </section>
+      ) : (
+        <div>
+          <label htmlFor="search-bar">Filter Items</label>
```

Comment:
Way to add a label element for the input!

Did you know that the checkboxes in this form are missing their labels? I get a warning about that when I audit the page with the Axe accessibility extension. You can learn more about it here: https://www.deque.com/axe/

Totally optional, but if you wanted to improve the page's accessibility you could also add labels for those checkboxes!